.github/workflows/ci.yaml

```diff
@@ -71,23 +72,34 @@ jobs:
            plugin-secrets-version: 4.0.0
            plugin-diff-version: 3.6.0
            extra-helmfile-flags:
+           v1mode:
          - helm-version: v3.10.3
            kustomize-version: v4.4.1
            plugin-secrets-version: 3.15.0
            plugin-diff-version: 3.5.0
            extra-helmfile-flags:
+           v1mode:
          - helm-version: v3.10.3
            kustomize-version: v4.5.7
            plugin-secrets-version: 4.0.0
            plugin-diff-version: 3.6.0
            extra-helmfile-flags:
+           v1mode:
+         # Helmfile v1
+         - helm-version: v3.10.3
+           kustomize-version: v4.5.7
+           plugin-secrets-version: 4.0.0
+           plugin-diff-version: 3.6.0
+           extra-helmfile-flags:
+           v1mode: "true"
```

Comment:
@yxxhero We now run the integration test suite for the v1 mode once per build, with the representative versions of helmfile dependencies, so that we won't break any v1 functionality until we finally stop maintaining v0.